lib/pypechain/pypechain/utilities/types.py

```diff
@@ -1,6 +1,9 @@
 """Utilities to help with Solidity types."""
 
 
+from logging import warn
```

Comment:
I think you want to do `logging.warning` here to use the logging setup everywhere else in the repo. Maybe warn is an alias for that? Either way, probably best to stay consistent with how we do it elsewhere.